_posts/developers/apps/inrupt-tutorial/2020-11-06-00_first-app.md

```diff
@@ -20,6 +20,8 @@ This tutorial creates an introductory application that uses Inrupt's
 [JavaScript Client Libraries](https://github.com/inrupt/solid-client-js) to read
 and write your name to your user profile.
 
+Note that while this tutorial leverages JavaScript, Solid is built on top of [open standards](https://github.com/solid/solid#standards-used). Any technical platform that implements the same standards can interact with Solid. For more information, see the [Solid Spec](https://github.com/solid/solid-spec).
```

Comment:
Hi @dynamoRando  -- Thanks for the PR to add a clarification.  I've added a suggestion so that we link to the page listing other language libraries instead of the Solid specifications, thinking that might further support new developers in getting started.  

```suggestion
Note:  Although this tutorial uses Inrupt's JavaScript libraries, you can use other libraries (be it for JavaScript or other languages) that support Solid. For more information, see the [Solid Developer Tools & Libraries](https://solidproject.org/developers/tools/).
```

Reply:
Sounds fair. Should we edit that [page](https://solidproject.org/developers/tools/) as well to note that Solid is built on open standards? I can submit a separate PR for that if need be.

I guess I'm hyper focused on highlighting that Solid is built on open standards because, speaking only for myself, I personally don't want developers to get discouraged if they don't see their favorite platform/framework listed. If I don't see my favorite platform X (PHP, Rust, etc) I don't want to give the impression that "well, Solid isn't ready for me yet" - because again, just as long as your platform has implementations for HTTP, OIDC, etc - you __can__ use Solid, it'll just take a little bit more effort.

Reply:
@dynamoRando - yes, please ping me or submit a PR on https://solidproject.org/developers/tools when you are ready to have your tool listed there.